dolphin: add page (#5833)
tldr-pages/tldr: pages/linux/dolphin.md

# dolphin

> KDE file manager.
> More information: <https://apps.kde.org/dolphin/>.

- Launch Dolphin:

`dolphin`

- Launch Dolphin and display a specific directory:

> `nautilus {{path/to/directory}}`

Reviewer: ```suggestion
`dolphin {{path/to/directory}}`
```

- Launch Dolphin with a specific file or directory selected:

`dolphin --select {{path/to/file_or_directory}}`

- Launch Dolphin in a separated window:

> `nautilus --new-window`

Reviewer: ```suggestion
`dolphin --new-window`
```

> - Launch Dolphin with a split view:

Reviewer 2: ```suggestion
- Launch Dolphin in split view:
```
Does this sound better?

Reviewer: Yeah, I like it better 👍🏻

`dolphin --split`

> - Start Dolphin daemon (only required for DBus interface):

Reviewer: ```suggestion
- Start the Dolphin daemon (only required to use the DBus interface):
```

> `nautilus --daemon`

Reviewer: ```suggestion
`dolphin --daemon`
```

- Display help:

> `nautilus --help`

Reviewer: ```suggestion
`dolphin --help`
```
